### PR TITLE
Add new cluster utilization item extension

### PR DESCRIPTION
**Pull request by rawagner**
We had similar extension point in the past but it was removed as nobody used it. Seems like we will need it soon for GPU plugin

Screenshot from 2021-11-22 10-12-33

### CODE REVIEW

File: frontend/packages/console-app/src/components/nodes/node-dashboard/utilization-popovers.tsx [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+  humanizeDecimalBytesPerSec,
+} from '@console/internal/components/utils';
+import { PodModel, ProjectModel } from '@console/internal/models';
+import ConsumerPopover, {
+  LimitsBody,
+} from '@console/shared/src/components/dashboard/utilization-card/TopConsumerPopover';
+import { NodeQueries, getTopConsumerQueries } from './queries';
+
+export const NodeUtilizationContext = React.createContext({
+  nodeIp: '',
```
Comment (vojtechszocs) [inline frontend/packages/console-app/src/components/nodes/node-dashboard/utilization-popovers.tsx:17]:
```suggestion
  nodeIP: '',
```

File: frontend/public/components/dashboard/dashboards-page/cluster-dashboard/utilization-popovers.tsx [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+import { useTranslation } from 'react-i18next';
+import { TopConsumerPopoverProps } from '@console/dynamic-plugin-sdk';
+import {
+  getTop25ConsumerQueries,
+  OverviewQuery,
+} from '@console/shared/src/promql/cluster-dashboard';
+import ConsumerPopover from '@console/shared/src/components/dashboard/utilization-card/TopConsumerPopover';
+import { ProjectModel, PodModel, NodeModel } from '../../../../models';
+import {
+  humanizeCpuCores,
```
Comment (vojtechszocs) [inline frontend/public/components/dashboard/dashboards-page/cluster-dashboard/utilization-popovers.tsx:11]:
```suggestion
  humanizeCPUCores,
```

File: frontend/packages/console-app/src/components/nodes/node-dashboard/utilization-popovers.tsx (hunk 1 above)
Comment (rawagner) [reply frontend/packages/console-app/src/components/nodes/node-dashboard/utilization-popovers.tsx:17]: fixed

File: frontend/public/components/dashboard/dashboards-page/cluster-dashboard/utilization-popovers.tsx (hunk 2 above)
Comment (rawagner) [reply frontend/public/components/dashboard/dashboards-page/cluster-dashboard/utilization-popovers.tsx:11]: this is widely used function across console code, not a new in this PR so i'd rather avoid changing it here.

**Comment by rawagner** [comment]
@yapei can you please take a look ? You can run demo plugin which should add `Foo` and `Bar` items to Cluster utilization card

**Comment by rawagner** [comment]
/label docs-approved
/label px-approved

We do not document SDK currently

**Comment by yapei** [comment]
verified that `Foo item`  and multi line utilization item `Bar item` can be viewed in `Cluster utilization` card
Screen Shot 2021-12-13 at 7 42 33 PM
/label qe-approved

**Pull request merged**